build_resolvers/lib/src/resolver.dart

```diff
@@ -138,40 +143,18 @@ class AnalyzerResolver implements ReleasableResolver {
 
       // Update the analyzer context with the latest sources
       _context.applyChanges(changeSet);
-      // Force resolve each entry point (the getter will ensure the library is
-      // computed first).
-      _entryLibraries = entryPoints.map((id) {
-        var source = sources[id];
-        if (source == null) return null;
-        var kind = _context.computeKindOf(source);
-        if (kind != SourceKind.LIBRARY) return null;
-        return _context.computeLibraryElement(source);
-      }).toList();
-    });
+    }).then((_) => this);
   }
 
   @override
   Stream<LibraryElement> get libraries {
-    if (_libraries == null) {
-      // Note: we don't use `lib.visibleLibraries` because that excludes the
-      // exports seen in the entry libraries.
-      _libraries = new Set<LibraryElement>();
-      _entryLibraries.forEach(_collectLibraries);
-    }
-    return new Stream.fromIterable(_libraries);
-  }
-
-  void _collectLibraries(LibraryElement lib) {
-    if (lib == null || _libraries.contains(lib)) return;
-    _libraries.add(lib);
-    lib.importedLibraries.forEach(_collectLibraries);
-    lib.exportedLibraries.forEach(_collectLibraries);
+    throw new UnimplementedError();
```

Comment:
Probably worth adding a comment about why this always throws (ditto below). Or maybe a single comment on the class saying why a couple of its methods throw.

Reply:
Done